internal/cmd/fingerprint/fingerprint.go

```diff
@@ -48,10 +49,9 @@ Special Terms | Meaning
 
 Example: 
 $ debricked files fingerprint . `+exampleFlags)
-
 	cmd.Flags().BoolVar(&shouldFingerprintCompressedContent, FingerprintCompressedContent, false, `Fingerprint the contents of compressed files by unpacking them in memory, Supported files: `+fmt.Sprintf("%v", fingerprint.ZIP_FILE_ENDINGS))
-
 	cmd.Flags().StringVar(&outputDir, OutputDirFlag, ".", "The directory to write the output file to")
+	cmd.Flags().IntVar(&minFingerprintContentLength, MinFingerprintContentLengthFlag, 0, "Set minimum content length (in bytes) for files to fingerprint.")
```

Comment:
we wanted this to be a greater than 0 by default :)

Reply:
Sure! But what, after some testing 35 bytes will be enough to excluded the version files from python where the content is `__version__=="x.x.x-flags"`, so should we go a bit higher than that for safety?

Reply:
Yes, let's go for 45